imports/plugins/core/graphql/server/no-meteor/ReactionError.js

```diff
@@ -12,5 +12,16 @@ export default class ReactionError extends ExtendableError {
     super(message);
     this.eventData = eventData;
```

Comment:
I believe @aldeed suggested that this file be moved out of the graphql directory...?

Reply:
ah, I made a copy of it and forgot to remove this one.